polaris-core/src/main/java/org/apache/polaris/core/config/FeatureConfiguration.java

```diff
@@ -234,4 +233,38 @@ public static void enforceFeatureEnabledOrThrow(
           .description("If true, the policy-store endpoints are enabled")
           .defaultValue(true)
           .buildFeatureConfiguration();
+
+  public static final FeatureConfiguration<Boolean> ALLOW_SPECIFYING_FILE_IO_IMPL =
+      PolarisConfiguration.<Boolean>builder()
+          .key("ALLOW_SPECIFYING_FILE_IO_IMPL")
+          .description(
+              "Config key for whether to allow setting the FILE_IO_IMPL using catalog properties. "
+                  + "Must only be enabled in dev/test environments, never in production systems.")
+          .defaultValue(false)
+          .buildFeatureConfiguration();
+
+  public static final FeatureConfiguration<Boolean>
+      ALLOW_INSECURE_STORAGE_TYPES_ACCEPTING_SECURITY_RISKS =
+          PolarisConfiguration.<Boolean>builder()
+              .key("ALLOW_INSECURE_STORAGE_TYPES_ACCEPTING_SECURITY_RISKS")
+              .description(
+                  "Allow usage of FileIO implementations that are considered insecure. "
+                      + "Enabling this setting exposes the service to SEVERE security risks, including "
+                      + "denial of service, corruption, data loss and more!"
+                      + "This must NEVER be set to 'true' for anything except tests! ")
+              .defaultValue(false)
+              .buildFeatureConfiguration();
+
+  public static final FeatureConfiguration<Boolean> INITIALIZE_DEFAULT_CATALOG_FILEIO_FOR_TEST =
+      PolarisConfiguration.<Boolean>builder()
+          .key("INITIALIZE_DEFAULT_CATALOG_FILEIO_FOR_TEST")
+          .description(
+              "Config key for initializing a default \"catalogFileIO\" that is available either via "
+                  + "getIo() or for any TableOperations/ViewOperations instantiated, via ops.io() "
+                  + "before entity-specific FileIO initialization is triggered for any such operations. "
+                  + "Typically this should only be used in test scenarios where a PolarisIcebergCatalog "
```

Comment:
If this is for tests I don't think it should be a FeatureConfiguration

Reply:
It's needed to _enable_ it _for_ tests.

Reply:
Why does it need to be a FeatureConfiguration to enable it for tests?

Reply:
The comment's outdated now, resolving.